app_test.go

```diff
@@ -1916,7 +1901,6 @@ func TestApp_Run_CommandHelpName(t *testing.T) {
 	}
 	cmd := &Command{
 		Name:        "foo",
-		HelpName:    "custom",
```

Comment:
Were there assertions around this `HelpName` value that were breaking, or did you need to remove it for some other reason? (or something else?)

Reply:
Yes. There seems to be a bug in the existing code(not this PR). Should we expect the help to have ? 

```
command custom bar - does bar things
```

Reply:
Updated the test to reflect actual usage